app/controllers/surveys_controller.rb

```diff
@@ -1,4 +1,58 @@
 class SurveysController < ApplicationController
   def send_survey
   end
+
+  def create
+    @survey = Survey.create!(survey_params)
```

Comment:
So is it convention when in the controller and we manipulate or access a resource for that controller, we always put it in an ivar? i.e. `send_survey` method, i'll put survey into `@survey`